Makefile

```diff
@@ -68,54 +71,57 @@ all: test manager
 
 # Run tests
 test: generate fmt vet manifests
-	go test ./pkg/... ./cmd/... -coverprofile $(COVER_FILE)
+	go test -v ./api/... ./controllers/... -coverprofile $(COVER_FILE)
 
 # Build manager binary
 manager: generate fmt vet
-	go build -o bin/manager ./cmd/manager/main.go
+	go build -o bin/manager main.go
 
-# Run using the configured Kubernetes cluster in ~/.kube/config
-run: generate fmt vet
-	go run ./cmd/manager/main.go
+# Run against the configured Kubernetes cluster in ~/.kube/config
+run: generate fmt vet manifests
+	go run ./main.go
 
 # Debug using the configured Kubernetes cluster in ~/.kube/config
 debug: generate fmt vet
-	dlv debug cmd/manager/main.go
+	dlv debug ./main.go
 
 # Run go fmt against code
 fmt:
-	go fmt ./pkg/... ./cmd/...
+	go fmt ./api/... ./controllers/...
 
 # Run go vet against code
 vet:
-	go vet ./pkg/... ./cmd/...
+	go vet ./api/... ./controllers/...
 
 ## --------------------------------------
 ## Deploying
 ## --------------------------------------
 
 # Install CRDs into a cluster
 install: $(KUSTOMIZE)
-	kubectl apply -k config/crds
+	kubectl apply -k config/crd
+
+# Uninstall CRDs from a cluster
+uninstall: $(KUSTOMIZE)
+	kubectl delete -k config/crd
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 deploy: $(KUSTOMIZE)
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
```

Comment:
PLease add back the kustomize edit here as well. Something similar to this:
cd config/manager && kustomize edit set image controller=${IMG}